From 5d905d79d5c6ac8fc8814d0baea1522f50ed48f9 Mon Sep 17 00:00:00 2001
From: Matt DeVillier <matt.devillier@puri.sm>
Date: Fri, 19 Jun 2020 17:02:22 -0500
Subject: [PATCH 2/3] soc/cannonlake/me.c: Print status regardless of device
 enable state

Checking the CSE device status before printing means it will skip
printing for devices with the ME disabled, leaving the user no easy
way to verify the ME is properly disabled. Remove the check.

Test: build/boot Librem Mini, verify ME status printed as expected
on device with disabled/neutered ME.

Change-Id: I939333199aa699039fec727beb094e4eb2ad7149
Signed-off-by: Matt DeVillier <matt.devillier@puri.sm>
---
 src/soc/intel/cannonlake/me.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/soc/intel/cannonlake/me.c b/src/soc/intel/cannonlake/me.c
index 7bbe1ae730..4fe5a96ade 100644
--- a/src/soc/intel/cannonlake/me.c
+++ b/src/soc/intel/cannonlake/me.c
@@ -103,9 +103,6 @@ void dump_me_status(void *unused)
 	union me_hfsts5 hfsts5;
 	union me_hfsts6 hfsts6;
 
-	if (!is_cse_enabled())
-		return;
-
 	hfsts1.data = me_read_config32(PCI_ME_HFSTS1);
 	hfsts2.data = me_read_config32(PCI_ME_HFSTS2);
 	hfsts3.data = me_read_config32(PCI_ME_HFSTS3);
-- 
2.25.1

